drivers/console/uart_console.c

```diff
@@ -602,7 +602,7 @@ static int uart_console_init(struct device *arg)
 #ifndef CONFIG_USB_DEVICE_AUTO_ENABLE
 	int ret;
 
-	ret = usb_enable();
+	ret = usb_enable(NULL);
```

Comment:
I believe all these need to be in the commit where the signature to `usb_enable()` changes so that the tree builds at every commit.